src/__tests__/patients/allergies/AllergiesList.test.tsx

```diff
@@ -11,57 +10,47 @@ import Allergy from '../../../shared/model/Allergy'
 import Patient from '../../../shared/model/Patient'
 
 describe('Allergies list', () => {
-  const setup = async (allergies: Allergy[]) => {
+  const setup = (allergies: Allergy[]) => {
     const mockPatient = { id: '123', allergies } as Patient
     jest.spyOn(PatientRepository, 'find').mockResolvedValueOnce(mockPatient)
     const history = createMemoryHistory()
     history.push(`/patients/${mockPatient.id}/allergies`)
-    let wrapper: any
-    await act(async () => {
-      wrapper = await mount(
+
+    return {
+      history,
+      ...render(
         <Router history={history}>
           <AllergiesList patientId={mockPatient.id} />
         </Router>,
-      )
-    })
-
-    wrapper.update()
-    return { wrapper: wrapper as ReactWrapper, history }
+      ),
+    }
   }
 
   it('should render a list of allergies', async () => {
     const expectedAllergies = [{ id: '456', name: 'some name' }]
```

Comment:
My suggestion is whatever makes sense. I generally stick with "mock" if I really want to differentiate if the variable or value is fake, however, I don't generally use expected mainly because the `expect()` tells you what is being asserted/expected

Reply:
@nobrayner ?

Reply:
Hmm... I was probably the one putting out most of the `expected*`.

The convention I follow is:
* If it mocks/is the mock of something, `mock*`
* If it is data I am using for assertions, `expected*`

I also use `expected*` even if it is the data being returned from a mock.

I've find this helps me reason with the code a lot easier, as it helps give expectation hints to me when I revisit later.